### ISSUE TITLE
Not supporting .NET Standard

### ISSUE THREAD

**Issue by rambotech** [MikeStall/DataTable#55]
This package is wonderful in .NET Framework.  Can it be made to build for .NET Standard 1.x/2.x?

**Comment by AlexKeySmith** [comment]
I agree, it would be fantastic and especially useful for Azure Functions V2.
